pkg/qliksense/context_configs.go

```diff
@@ -75,7 +75,7 @@ func (q *Qliksense) processSecret(ra *api.ServiceKeyValue, rsaPublicKey *rsa.Pub
 		secretFolder := qliksenseCR.GetK8sSecretsFolder(q.QliksenseHome)
 		secretFileName := filepath.Join(secretFolder, ra.SvcName+".yaml")
 
-		secretName = fmt.Sprintf("%s.%s.%s", qliksenseCR.GetName(), ra.SvcName, "senseinstaller")
+		secretName = fmt.Sprintf("%s-%s-%s", qliksenseCR.GetName(), ra.SvcName, "senseinstaller")
```

Comment:
Just curious: %s.%s.%s would have worked too, isn't it?

Reply:
yea, but putting secret name with dot notation is non-conventional